examples/pages/misc_examples.py

```diff
@@ -70,4 +70,4 @@
 
 with st.echo():
     # call to render Folium map in Streamlit
-    st_folium(m, width=2000, height=500, returned_objects=[])
+    st_folium(m, width=2000, height=500, returned_objects=[], debug=True)
```

Comment:
Why is `debug=True` necessary all of the sudden?

Reply:
It's not necessary, just useful for debugging. I could remove all of them, but I thought I wouldn't hurt to leave them in